src/prefect/tasks/docker/images.py

```diff
@@ -119,8 +120,27 @@ def run(
             raise ValueError("A repository to pull the image from must be specified.")
 
         client = docker.APIClient(base_url=docker_server_url, version="auto")
+        self.logger.debug(
+            "Starting docker pull for repository {repo} with tag {tag}...".format(
+                repo=repository, tag=tag
+            )
+        )
+        feed = client.pull(repository=repository, tag=tag, stream=True)
```

Comment:
The output logs from the call when `stream=True`, aren't identical to those being output when `stream=False`. Should we be worried about backwards compatibility issues? If so, we can log the output from `stream=False` and just return that?

Reply:
Hmm yeah this is an interesting one. I'm not sure if the best way to add logs here would be to actually log the output of `stream=True` or add a log before the `.pull` what says something along the lines of `"Pulling {}:{}".format(repository, tag)`. This is because the pull stream output is quite massive and would add a lot of noise to the logs. I vote for a simple `Pulling` log before the call and changing the return to be the output of the stream.